src/include/wiredtiger.in

```diff
@@ -1961,6 +1948,27 @@ struct __wt_session {
 	 */
 	int __F(checkpoint)(WT_SESSION *session, const char *config);
 
+	/*!
+	 * Reset the snapshot.
```

Comment:
Why move to here?

Reply:
As far as I can tell, the order was random before; so, I split it up into 3 groups: "transactions" (so, begin, commit, prepare, rollback), "timestamps" (so, query_timestamp and timestamp_transaction), and "support" (checkpoint, reset_snapshot and transaction_pinned_range). I'm open to any ordering you like better, but what we had didn't make sense to me.